src/html/scan.js

```diff
@@ -375,15 +399,37 @@ function updateConfig(data, options) {
       _('sbus-config').style.display = 'none';
     }
   }
+
+  _('serial1-protocol').onchange = () => {
+    if (_('is-airport').checked) {
+      _('rcvr-uart-baud').disabled = false;
+      _('rcvr-uart-baud').value = options['rcvr-uart-baud'];
+      _('serial1-config').style.display = 'none';
+      _('sbus-config').style.display = 'none';
+      return;
+    }
+  }
+
   updatePwmSettings(data.pwm);
   _('serial-protocol').value = data['serial-protocol'];
   _('serial-protocol').onchange();
+  _('serial1-protocol').value = data['serial1-protocol'];
+  _('serial1-protocol').onchange();
   _('is-airport').onchange = _('serial-protocol').onchange;
```

Comment:
`is_airport` onchange should be function that now calls both serial and serial1 onchange functions.

Reply:
is_airport: done
upgrader: done, but not sure about this. initializing serial1 protocol should be enough, right?